layers/ports.go

```diff
@@ -106,3 +106,12 @@ func (a UDPLitePort) String() string {
 	}
 	return strconv.Itoa(int(a))
 }
+
+func init() {
```

Comment:
instead of doing this in init, maybe build udpPortLayerType with this already in place?

```go
var dupPortLayerType = [65536]gopacket.LayerType{
  53: LayerTypeDNS,
  123: LayerTypeNTP,
  ...
}
```